### DISCUSSION TITLE
Allow empty line after hashbang.

### DISCUSSION THREAD

**Discussion by latin-1** [rome/tools#2324] [Suggestions]
```js
#!/usr/bin/env node

console.log(1);
```

https://play.rome.tools/?lineWidth=80&indentStyle=tab&indentWidth=2&typescript=false&jsx=false#IyEvdXNyL2Jpbi9lbnYgbm9kZQoKY29uc29sZS5sb2coMSk7

**Comment by ematipico** [comment]
This is definitely a bug! Thank you for reporting
